src/server/service/debug.rs

```diff
@@ -409,6 +409,46 @@ impl<T: RaftStoreRouter + 'static> debugpb::Debug for Service<T> {
 
         self.handle_response(ctx, sink, f, TAG);
     }
+
+    fn get_store_info(
+        &mut self,
+        ctx: RpcContext<'_>,
+        _req: GetStoreInfoRequest,
+        sink: UnarySink<GetStoreInfoResponse>,
+    ) {
+        const TAG: &str = "debug_get_store_id";
+        let debugger = self.debugger.clone();
+
+        let f = self.pool.spawn_fn(move || {
+            let mut resp = GetStoreInfoResponse::default();
+            if let Ok(store_id) = debugger.get_store_id() {
+                resp.set_store_id(store_id);
```

Comment:
`debugger::get_store_id` returns a `Result<u64>` , could you hanlde the error here like?

Reply:
What I can do with the error is setting a zero value. If not handle this, the default store id will be 0. The results in `tikv-ctl` are same.